### PR TITLE
Remove hard-coded lists of collections (fixes #26)

### LINKED ISSUE #26: The validate_signature function should take collections list from changes endpoint
Why would we want a different list of collections than the one used by Firefox?

It would avoid us doing https://github.com/mozilla-services/cloudops-deployment/pull/1580 (we want remote settings self-service in the end)

### PR DESCRIPTION
**Pull request by leplatrem**
Fixes #26

### CODE REVIEW

File: aws_lambda.py [hunk 1]
```diff
@@ -56,6 +64,8 @@ def validate_signature(event, context):
 
         # 1. Grab collection information
         dest_col = client.get_collection()
+        if 'signature' not in dest_col['data']:
+            continue
```
Comment (glasserc) [inline aws_lambda.py]: Hang on, doesn't this sort of defeat the purpose of the lambda (by allowing unsigned collections to go through)?

File: aws_lambda.py [hunk 2]
```diff
@@ -143,13 +129,45 @@ def timestamp_to_date(timestamp_milliseconds):
...
+    print('Looking at %s: ' % client.get_endpoint('collection'))
+    changes = client.get_records()
+
+    # Look at the signer configuration on the server.
+    info = client.server_info()
+    signed_resources = info['capabilities']['signer']['resources']
+
+    def get_source(change):
+        # Small helper to pick the source collection from the collection
+        # mentionned in the changes endpoint. (eg. blocklists/plugins -> staging/plugins)
```
Comment (glasserc) [inline aws_lambda.py]: `mentioned`. Also, I would probably say `get` instead of `pick`, or maybe `find` or `identify`.

File: aws_lambda.py [hunk 3]
```diff
@@ -208,27 +196,30 @@ def schema_updater(event, context):
...
+    # AMO schemas are all in the staging bucket.
+    # See https://github.com/mozilla-services/cloudops-deployment/blob/dc72e8241f5f721e49c054c8726a4fc4a7089b61/projects/kinto-lambda/ansible/playbooks/schema_updater_lambda.yml#L16-L20
+    bucket = event.get('bucket', 'staging')
 
     # Open the file
-    with codecs.open("schemas.json", 'r', encoding='utf-8') as f:
+    with codecs.open('schemas.json', 'r', encoding='utf-8') as f:
         schemas = json.load(f)['collections']
 
     # Check all collections
```
Comment (glasserc) [inline aws_lambda.py]: Maybe update this comment?

File: aws_lambda.py (hunk 1 above)
Comment (leplatrem) [reply aws_lambda.py]: haha, yes indeed. Instead of this, I should probably check the signer capabilities!

**Pull request merged**